mm/iob/Kconfig

```diff
@@ -32,6 +32,18 @@ config IOB_BUFSIZE
 		chain.  This setting determines the data payload each preallocated
 		I/O buffer.
 
+config IOB_ALIGNMENT
+	int "Alignment size of each I/O buffer"
+	default 1
```

Comment:
Please check if 1 or 2 should be default. The low level stack assumes 2 bytes alignment, but I'm not sure what goes into IOB

Reply:
We have a better solution, discard this PR. Please refer to https://github.com/apache/incubator-nuttx/pull/6834